cypress/integration/login.js

```diff
@@ -1,27 +1,37 @@
+import LoginPage from '../objects/LoginPage';
+import TimeLoggin from '../objects/TimeLoggin';
+const timeLoggin = new TimeLoggin();
+const loginPage = new LoginPage();
+
+
 describe('Sourcebooks login', function() {
 
-    it('Should display validation for empty user after attempted loggin', function () {
-        
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+    beforeEach(function() {
+        cy.loginAs("Admin")
+        cy.visit('/')
     })
+    /*it('Should display validation for empty user after attempted loggin', function () {
+        
+        loginPage.visit()
+        loginPage.getUserValidationIndicator().should('not.visible')
+        loginPage.getSubmitButton().click();
+        loginPage.getUserValidationIndicator().should('be.visible')
+    })*/
 
     it('Should be able to login with role User', function () {

```

Comment:
Now it is not actually testing User role, it is admin. This loginAs functionality should be used in test case 3 file to avoid using login each time

Reply:
also this test case is covered in T2.js file

Reply:
this is a duplicate test case of T2.js
it('Should be able to login with user Jaunius Mykolaitis to role User',